### PR TITLE
ECR Modules

### PR DESCRIPTION
**Pull request by frdvo**
ECR Module is here, by default it is not public. Check the instructions on readme.

### CODE REVIEW

File: terraform/modules/ecr/README [hunk 1]
```diff
@@ -0,0 +1,28 @@
+DevOps Academy Project 3 Jun 2020 ECR Repo
+
+Instructions:
+
+1. After apply terraform get the repository URL
+
+terraform output | grep repository_url
```
Comment (jayanath) [inline terraform/modules/ecr/README]: Great job 🚀 just one comment. Is there a possibility for us to use the outputs.tf to return the repository URL and use it in the other template as `module.<MODULE NAME>.<OUTPUT NAME>`
Have a look here to see if its possible. [https://www.terraform.io/docs/configuration/outputs.html](url)

Comment (frdvo) [reply terraform/modules/ecr/README]: I've written the output accordingly to this documentation. I suggested use grep to filter the repository URL on terraform output command.

Comment (frdvo) [reply terraform/modules/ecr/README]: @jayanath I improved it using jq to filter the repository variable value. I will use it on docker Makefile

File: terraform/modules/ecr/README [hunk 2]
```diff
@@ -4,11 +4,11 @@ Instructions:
 
 1. After apply terraform get the repository URL
 
-terraform output | grep repository_url
+terraform output -json ecr-module | jq .ecr | jq .repository_url
```
Comment (jayanath) [inline terraform/modules/ecr/README:7]: Not sure if we should add another dependency? but I think we will have to refactor our code a bit anyways. So lets' get this merged for now

**Review by jayanath — APPROVED** [review]
Great stuff. Let's get this merge.

**Pull request merged**